markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+---
+title: "Metadata Template"
+author: "Christian Krestel"
+date: "2023-06-27"
+output:
+  pdf_document:
+    includes:
+      in_header: "preamble.tex"
+    toc: true
+    toc_depth: 3
+    latex_engine: xelatex
+---
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Version number: 3
+
+# Template: Code repository catalogue
+
+1. Create new project: Type script
+2. Screenshot of the \emph{complete} first page
+3. For a published paper, prepare the link to the repository. For a WP prepare a zip that contains all data and code; For \textbf{DIME verified} packages also the reproducibility verification statement
+
+# Information on metadata
+This is information about the creator of the metadata. Sufficient fields
+\begin{enumerate}
+  \item Name: Second name only: Smith
+  \item Abbreviation: PS
+  \item Name: Smith
+  \item Affiliation: World Bank
+  \item Production date: Year-Month-Day
+  \item Version number: Entered as 1,2,3
+\end{enumerate}
+
+# Project description
+## Title statement
+Sufficient fields are
+\begin{enumerate}
+  \item Primary ID: Ideally a DOI or an organization consistent scheme;
+      \begin{enumerate}
+        \item Published Paper: 
+            \begin{enumerate}
+              \item A corresponding PRWP exists: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+                  Example: PP_CRI_2018_PRWP-8524_v01
+                \end{lstlisting}
+              \item A corresponding PRWP does not exit: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_EXT-EXTno_versionno
+                  Example: PP_CRI_2018_EXT-1_v01
+                \end{lstlisting}
+                    EXTno is a unique running number assigned by Dime.
+            \end{enumerate}
+        \item Working Paper: FORMAT
+          \begin{lstlisting}
+            RR_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+            RR_CRI_2018_PRWP-8525_v01
+          \end{lstlisting}
+        \end{enumerate}
+  \item Other identifiers: DOI and https://doi.org/number
```

Comment:
Just a note that we still need to work out the exact protocol for generating and entering the DOIs for packages that we publish directly. DECDG owns the account for generating DOIs). they use the metadata from NADA to generate the DOI, so the sequence will, i think, need to be that we create the catalog entry, share the draft with someone at DG, they run a script to harvest the metadata and create the DOI, and then they come back to us with the DOI (or directly enter it into the catalogue). I've raised the issue with the DG team and will update, so let's leave this flagged as requiring a later update for now.

Reply:
Ok. We have a primary ID, the DOI can be added later